packages/amplify-storage-simulator/src/server/S3server.ts

```diff
@@ -0,0 +1,327 @@
+import * as express from 'express';
+import * as cors from 'cors';
+import { join ,normalize} from 'path';
+import { createServer, request } from 'http';
+import {readFile, unlink, readdirSync, statSync, ensureFileSync, writeFileSync ,existsSync} from 'fs-extra';
+import * as xml from 'xml';
+import * as bodyParser from 'body-parser';
+import * as convert from 'xml-js';
+import * as e2p from 'event-to-promise';
+import * as serveStatic from "serve-static";
+import * as glob from 'glob';
+import * as o2x from 'object-to-xml';
+import * as uuid from 'uuid';
+import * as md5 from 'md5-file';
+import * as etag from 'etag';
+
+import { StorageSimulatorServerConfig } from '../index';
+
+
+var corsOptions = {
+  maxAge: 20000,
+  exposedHeaders: ['x-amz-server-side-encryption', 'x-amz-request-id', 'x-amz-id-2', 'ETag']
+}
+export class StorageServer {
+  private app;
+  private server;
+  private connection;
+  private route; // bucket name get from the CFN parser
+  url: string;
+  private uploadId;
+  private localDirectoryPath: string;
+
+  constructor(
+    private config: StorageSimulatorServerConfig,
+  ) {
+    this.localDirectoryPath = config.localDirS3;
+    //console.log("path file", this.localDirectoryPath);
+    this.app = express();
+    this.app.use(express.json());
+    this.app.use(cors(corsOptions));
+    //this.app.set('etag', false);
```

Comment:
Remove the commented code unless you are documenting something

Reply:
removed